### ISSUE TITLE
Manta

### ISSUE THREAD

**Issue by HellDragon-HK** [GreatEmerald/UT3Vehicles#9]
Issues and notes for the Manta from in-game testing

UT2004 Draw  - 1.0?
UT3    Draw  - 1.0?
UT3Port Draw - 1.0 
Collision seems good

Looks longer than the UT3 Manta but I believe that's my resolution doing that which UT2004 has some wierd issues with resolutions, I even had it where the ground had holes in it when I moved around (confirmed it was the resolution that caused that because changing the in-game resolution and it stopped happening), I feel it's important to notice this because it would also be true with the other vehicles

  - [Priority]  
  - [Secondary]
  - [Orange engine flames are missing, possibly not orange on blue team, hadn't thought to check that]  
  - [Team colored lights on the front sides (in that brown or black area toward the front) is missing]  
  - [The sound you hear when actually moving forward is missing on the port, all I hear is the rotor blades]
  - [Extra]  
  - [I think the UT3 Manta turns half a second faster than the port]  
  - [UT3 primary fire sound I think is slightly different]  
  - [Headlight appears on but no light is actually being emmited, I can't see it hitting a wall for example, stock UT2004 Manta seems to be this way as well though]

**Comment by HellDragon-HK** [comment]
Impact results for Manta

Manta Impact with Manta
-------UT3 Results--------
--Front Hit   - Fair knockback and slight turn, easily pushed around
--Side Hit    - Little knockback and Manta goes over Manta, easily pushed around
--Back Hit    - Fair knockback and slight turn, easily pushed around
--Diagnol Hit - Little knockback Manta gets spun almost 90 degrees, easily pushed around
------UT2004 Results------ 
--Front Hit   - Little knockback, Manta goes over Manta when trying to push
--Side Hit    - No movement, Manta goes over Manta when trying to push but can be slightly pushed
--Back Hit    - Little knockback and gets turned 90 degrees, Manta goes over Manta but not before getting a fair push
--Diagnol Hit - Fair knockback, Manta goes over Manta when trying to push
Extra Note: Both Mantas in UT2004 took damage and the ones in UT3 did not or was very little

Manta Impact with Scorpion
-------UT3 Results--------
--Front Hit   - Little knockback and causes Scorpion to go on it's back wheels for a second, tires squeel, cannot be pushed
--Side Hit    - Little knockback and causes Scorpion to nearly flip or completely flip on it's side, tires squeel, cannot be pushed
--Back Hit    - Little knockback and causes Scorpion to go on it's front wheels for a second, tires squeel, cannot be pushed
--Diagnol Hit - Little knockback but stays on the ground, tires squeel, cannot be pushed
------UT2004 Results------ 
--Front Hit   - Manta goes over Scorpion, can be slowly pushed
--Side Hit    - Little knockback and causes Scorpion to nearly flip or completely flip on it's side, cannot be pushed
--Back Hit    - Manta goes over Scorpion or little knockback and Scorpion's back wheels are lifted off the ground, cannot be pushed
--Diagnol Hit - Little knockback and side that was hit gets lifted up a little, cannot be pushed
Extra Note: Manta took damage in UT2004 but not in UT3, Scorpion was not damaged in either game

Manta Impact with Hellbender
-------UT3 Results--------
--Front Hit   - Little knockback and Hellbender is lifted off it's front wheels, tires squeel, cannot be pushed
--Side Hit    - Little knockback and Hellbender is flipped on it's side, tires sometimes squeel, cannot be pushed
--Back Hit    - Little knockback and Hellbender is lifted off it's back wheels, tires squeel, cannot be pushed
--Diagnol Hit - Fair knockback but no lift up, cannot be pushed
------UT2004 Results------ 
--Front Hit   - same as UT3 except can be slowly pushed and no tire squeel
--Side Hit    - same as UT3 except no tire squeel (cannot be pushed)
--Back Hit    - Same as UT3 except can be slowly pushed and no tire squeel
--Diagnol Hit - Same as UT3 except can be slowly pushed
Extra Note: Manta took damage in UT2004 but not in UT3, Hellbender was not damaged in either game, and impact sounds in UT2004 are different

Manta Impact with Stealthbender
-------UT3 Results--------
--Front Hit   - same as hitting the Hellbender
--Side Hit    - same as hitting the Hellbender including being flipped over
--Back Hit    - same as hitting the Hellbender
--Diagnol Hit - same as hitting the Hellbender except it almost flipped which probably can happen with the Hellbender as well
------UT2004 Results------ 
--Front Hit   - pending installing a swap tool
--Side Hit    - pending installing a swap tool
--Back Hit    - pending installing a swap tool
--Diagnol Hit - pending installing a swap tool
Extra Note: No damage to either vehicle in UT3

Manta Impact with Raptor
-------UT3 Results--------
--Front Hit   - Little knockback, can be fairly pushed
--Side Hit    - Little knockback and causes little spin, can be fairly pushed
--Back Hit    - Little knockback and causes little spin, can be fairly pushed
--Diagnol Hit - Little knockback and causes little spin, turns more than gets pushed
------UT2004 Results------ 
--Front Hit   - Fair knockback, can be pushed fairly but acts like it's on ice
--Side Hit    - Fair knockback, can be pushed fairly but acts like it's on ice
--Back Hit    - Fair knockback, can be pushed fairly but acts like it's on ice
--Diagnol Hit - Fair knockback and is spun around quite a bit, can be pushed fairly but acts like it's on ice and spins
Extra Note: Raptor in UT2004 took a lot of damage and Manta took some, neither vehicle in UT3 was damaged

Manta Impact with Cicada
-------UT3 Results--------
--Front Hit   - Almost no knockback but turns slightly, cannot be pushed
--Side Hit    - Almost no knockback, cannot be pushed
--Back Hit    - Little knockback, cannot be pushed
--Diagnol Hit - Only seems to turn, cannot be pushed
------UT2004 Results------ 
--Front Hit   - Little knockback, cannot be pushed
--Side Hit    - Little knockback, cannot be pushed
--Back Hit    - Little knockback, cannot be pushed
--Diagnol Hit - Little knockback and spin, cannot be pushed but can be spun
Extra Note: Cicada and Manta took damage in UT2004 but not in UT3

Manta Impact with Hellfire SPMA
-------UT3 Results--------
--Front Hit   - Little knockback, cannot be pushed
--Side Hit    - Little knockback and slightly tries to flip, cannot be pushed
--Back Hit    - Little knockback, cannot be pushed
--Diagnol Hit - Almost no knockback, cannot be pushed
------UT2004 Results------ 
--Front Hit   - Little knockback but SPMA is lifted off it's front wheels, can be slowly pushed
--Side Hit    - Little knockback but easily flipped on it's side, cannot be pushed
--Back Hit    - Little knockback but SPMA is lifted off it's treads, can be slowly pushed
--Diagnol Hit - Little knockback, cant be slowly pushed
Extra Note: Manta was damaged in UT2004 and SPMA was not, neighter vehicle was damaged in UT3, SPMA tread section wheels get messed up after a side hit and do not fix automatically, getting into the SPMA instantly fixes them though

Manta Impact with Paladin
-------UT3 Results--------
--Front Hit   - No knockback, front wheels are very very slightly lifted off the ground, cannot be pushed
--Side Hit    - No knockback and very slightly is lifted off the ground, cannot be pushed
--Back Hit    - No knockback, back wheels are very slightly lifted off the ground, cannot be pushed
--Diagnol Hit - No knockback, cannot be pushed
------UT2004 Results------ 
--Front Hit   - No knockback, front wheels are lifted off the ground about the same as UT3, cannot be pushed and is not lifted into the air
--Side Hit    - No knockback but completely flipped upside down, cannot be pushed but a slight tap lifts the Paladin into the air
--Back Hit    - No knockback, back wheels lifted high off the ground, cannot be pushed but a slight tap lifts the Paladin into the air
--Diagnol Hit - Little knockback and Paladin jumps into the air on top of the Manta, cannot be pushed but a slight tap lifts the Paladin into the air
Extra Note: Manta and Paladin are both taking damage in UT2004 but not in UT3

Manta Impact with Goliath
-------UT3 Results--------
--Front Hit   - No knockback, can be very slowly pushed
--Side Hit    - No knockback, cannot be pushed
--Back Hit    - No knockback, cannot be pushed
--Diagnol Hit - No knockback, cannot be pushed
------UT2004 Results------ 
--Front Hit   - Little knockback, can be slowly pushed
--Side Hit    - Little knockback (and I thought I saw it try to flip), can be slowly pushed
--Back Hit    - Little knockback, can be slowly pushed
--Diagnol Hit - Little knockback, cane be slowly pushed and quickly turned
Extra Note: Manta took a lot of damage and Golitath was not damaged, neither vehicle in UT3 was damaged

Manta Impact with Leviathan
-------UT3 Results--------
--Front Hit   - No knockback, Manta has damage indicators, cannot be pushed
--Side Hit    - No knockback, Manta has damage indicators, cannot be pushed
--Back Hit    - No knockback, no damage indicators, cannot be pushed
--Diagnol Hit - No knockback, no damage indicators, cannot be pushed
------UT2004 Results------ 
--Front Hit   - Little knockback, can be very slowly pushed
--Side Hit    - Little knockback, cannot be pushed
--Back Hit    - Fair knockback, cannot be pushed
--Diagnol Hit - Fair knockback, can be very slowly pushed
Extra Note: Manta and Levitathan took damage in UT2004, in UT3 only Manta took damage

Manta Impact with Viper
-------UT3 Results--------
--Front Hit   - Little knockback and front is lifted up, can be pushed
--Side Hit    - Little knockback and slightly tries to flip, easily pushed
--Back Hit    - Fair knockback, can be pushed
--Diagnol Hit - Fair knockback, can be pushed
------UT2004 Results------ 
--Front Hit   - pending installing a swap tool
--Side Hit    - pending installing a swap tool
--Back Hit    - pending installing a swap tool
--Diagnol Hit - pending installing a swap tool
Extra Note: In UT3 vehicles did not take damage

Manta Impact with Scavenger
-------UT3 Results--------
--Front Hit   - Fair knockback and is slightly lifted into the air, easily pushed and legs move
--Side Hit    - Fair knockback, easily pushed and legs move
--Back Hit    - Little knockback, easily pushed and legs move
--Diagnol Hit - Little knockback, easily pushed and legs move
------UT2004 Results------ 
--Front Hit   - pending being added to mod
--Side Hit    - pending being added to mod
--Back Hit    - pending being added to mod
--Diagnol Hit - pending being added to mod

Manta Impact with Nightshade
-------UT3 Results--------
--Front Hit   - Little knockback, easily pushed
--Side Hit    - Little knockback, easily pushed
--Back Hit    - Little knockback, easily pushed
--Diagnol Hit - Little knockback and slightly tries to flip, spins rather than being pushed
------UT2004 Results------ 
--Front Hit   - pending installing a swap tool
--Side Hit    - pending installing a swap tool
--Back Hit    - pending installing a swap tool
--Diagnol Hit - pending installing a swap tool

Manta Impact with Fury
-------UT3 Results--------
--Front Hit   - Little knockback, cannot be pushed
--Side Hit    - Fair knockback and spins 90 degrees, cannot be pushed
--Back Hit    - Little knockback and is lifted off the ground, cannot be pushed
--Diagnol Hit - Little knockback and slightly spins
------UT2004 Results------ 
--Front Hit   - pending installing a swap tool
--Side Hit    - pending installing a swap tool
--Back Hit    - pending installing a swap tool
--Diagnol Hit - pending installing a swap tool

Manta Impact with Nemesis
-------UT3 Results--------
--Front Hit   - No knockback but slightly lifts off the ground, Manta is damaged
--Side Hit    - No knockback, Manta sometimes gets damaged
--Back Hit    - No knockback
--Diagnol Hit - No knockback
------UT2004 Results------ 
--Front Hit   - pending being added to mod
--Side Hit    - pending being added to mod
--Back Hit    - pending being added to mod
--Diagnol Hit - pending being added to mod

Manta Impact with Darkwalker
-------UT3 Results--------
--Front Hit   - No knockback, Manta sometimes damaged, cannot be pushed
--Side Hit    - No knockback, can be turned and legs move each time
--Back Hit    - Little knockback and turning and also causes legs to move, canot be pushed
--Diagnol Hit - No knockback but caused slight turn, can be turned and legs move each time
------UT2004 Results------ 
--Front Hit   - pending being added to mod
--Side Hit    - pending being added to mod
--Back Hit    - pending being added to mod
--Diagnol Hit - pending being added to mod

**Comment by GreatEmerald** [comment]
The way vehicle sounds work in UT2004 is that the "engine" sound (idle one) is played on idle, and when the vehicle starts moving, the engine sound simply gets louder and higher pitched. It should be possible to override this, but it involves writing new code for the sound handling.

The dust in UT2004 is per-level, there is a map setting of what colour the dust should be. That's better than the UT3 system anyway.

Looks like the impulse differences with the manta are not that big either, good to know.

**Comment by HellDragon-HK** [comment]
Ok I can agree on the dust blow, I see you checked it but I don't see a line through it is that a browser glitch on my side?

I'm posting my Paladin test results now btw

**Comment by GreatEmerald** [comment]
Nah, I'm just being lazy. Checking it off is a matter of clicking, but also adding a strikethrough effect requires editing the post and adding `~` symbols before and after the point that's striken through. It doesn't matter a whole lot.

**Comment by HellDragon-HK** [comment]
Ok so any check marks I see that don't have that you want me to try and add it to save you the trouble?
Well we now know you have one of those Progammer's virtues lol

Btw did you get my e-mails I sent you that I would have put here but didn't know where they should go?

**Comment by GreatEmerald** [comment]
I'm going through the whole list very slowly, then I'll go through the emails.

**Comment by HellDragon-HK** [comment]
Update on the Manta movement sound missing, it works in 32bit but not 64bit, any idea as to why?

**Comment by HellDragon-HK** [comment]
I just noticed in UT3 the Manta is immune to collision damage except when jumping, in UT2004 it gets hurt from any tiny scrape of an object...any chance to replicate that ability?

**Comment by 100GPing100** [comment]
@HellDragon-HK Yes this is possible to replicate, as long as it's confirmed this is how it works it's easy to implement.

**Comment by HellDragon-HK** [comment]
I'm pretty positive about it, but I can double check if you need first.

**Comment by HellDragon-HK** [comment]
Trying to add the engine flames which this should do something shouldn't it?

	TrailEffectPositions(0)=(X=-148,Y=-26,Z=51);
   	TrailEffectPositions(1)=(X=-148,Y=26,Z=51);
	TrailEffectClass=class'Onslaught.ONSAttackCraftExhaust'

It doesn't seem to do anything though  I know the position would need to be adjusted but it isn't even appearing so far.

**Comment by HellDragon-HK** [comment]
@GreatEmerald 
I've edited the Manta's collision so it sits on the front metal bars like it does in UT3 and tested it in-game on ONS-Dawn, ONS-Torlan, & ONS-CE-DryIce and it seems to handle the same without any side effects including same amount of ground collision damage taken on hills.

I don't know how to create a pull request for a ukx file.

**Comment by Unre-Alex** [comment]
In UT3 the Manta is higher from the ground but not in a position like that.

Add the gun barrels move in UT3.

**Comment by HellDragon-HK** [comment]
You're right, it's flat, I'll see about editing it again to get that, and also right about the gun barrel movement which appears to be like the Raptor up, down, and to side,

**Comment by HellDragon-HK** [comment]
@Unre-Alex 
Ok try this one
https://drive.google.com/open?id=1sgwvAzaayLAR5HKbgiuiqrEZAF0VKmBU

**Comment by Unre-Alex** [comment]
Right, it doesn't work as it should. Did you edit the modified file or the old one?

**Comment by HellDragon-HK** [comment]
Changed the one that was already changed yes

**Comment by HellDragon-HK** [comment]
@Unre-Alex 
Ok try this one please, it's not spawning stuck on Urban now anywhere so it might be good
https://drive.google.com/open?id=1DXgZEsNuLL_pNx_Q-bwvYUPC2oSqqYdb

**Comment by Unre-Alex** [comment]
But the thing is why it randomly spawns at different heights...

**Comment by HellDragon-HK** [comment]
I don't know, all I did was add a collision box along the bottom length and mostly width of it and changed the z translation from 80 to 70

Are you still having issues with that file as well, what map?

**Comment by Unre-Alex** [comment]
Doesn't matter, it can not be edited like that apparently.

**Comment by HellDragon-HK** [comment]
I don't understand, it's spawning right in every map I've tried and not at random heights, the only one that is questionable is Red Base on Torlan and it's always in the same spot every time I've watched it spawn.

As long as it isn't stuck it shouldn't be a problem should it?  Also are trying the 3rd file because I didn't change z translation before then so the new box was spawning in the ground along with those bar section.

**Comment by HellDragon-HK** [comment]
I was on Urban and I didn't see that but I'll look again

**Comment by HellDragon-HK** [comment]
@Unre-Alex 
Ok I think I fixed it by removing the new box and changing z translation to 68
https://drive.google.com/open?id=13YoAtFz4ftBbuo0IgDSYCO1PtwmH9ZbS

Arctic Stronghold and Urban Mantas appear to be spawning at the same height now and without any strange bouncing, needs further testing to be sure though.

**Comment by Unre-Alex** [comment]
Now it's better. Looking now in the editor, it looks like the Manta's spawn height has to do with how the default Manta is placed in a map.

**Comment by HellDragon-HK** [comment]
Isn't map placement true for every vehicle's spawn height?

**Comment by HellDragon-HK** [comment]
@Unre-Alex 
Does the damage in UT2004 seem a little off to you, in UT3 it seems to take exactly 7 shots to kill another Manta and in UT2004 it usually seems to take 6 and sometimes 8 but never 7

**Comment by Unre-Alex** [comment]
In UT3 it takes 8 and in UT2k4 takes 6, and in UT3 the projectiles from each gun barrel shoot at the exact same spot while in UT2k4 they don't.

**Comment by HellDragon-HK** [comment]
It takes 7 with direct shots on the Manta's front that is sitting at it's spawn on Suspense, always 7 :S
You're right about the projectile spread, it's harder to hit something in 2004

**Comment by Unre-Alex** [comment]
No it takes always 8 on Suspense and other maps, no mutators, different difficulty levels, all angles.

**Comment by HellDragon-HK** [comment]
Difference in PS3 maybe because it's always 7 directly on the front gun barrel area, I'll have to try UT3 PC and see if I get what you get....could be PS3 isn't patched the same?

**Comment by Unre-Alex** [comment]
man don't bring ps3 into this project.

**Comment by HellDragon-HK** [comment]
I'm doing most of my testing with PS3 though so I can look at both at the same time, I just find it interesting it's not identical in damage values

**Comment by HellDragon-HK** [comment]
@Unre-Alex Could you try this and see what you think?
https://drive.google.com/open?id=1ArnfWnadLlh4Da-tdRWKjJHw5dMBNTX8
Changes-
Rolling seems less intense and more like UT3 is hopefully
Ground Speed is UT3 value
Driver Position improved by bringing up and closer to the controls

**Comment by Unre-Alex** [comment]
It's not good, it feels heavy now. It does roll a little in UT3 and the air control in much different in UT3 as well.

**Comment by HellDragon-HK** [comment]
I can agree with the heavy feeling so I'll keep experimenting, the only values I'm changing for the tilt are RollTorqueMax and RollDamping

**Comment by HellDragon-HK** [comment]
@Unre-Alex Ok try this and see what you think, roll & driver position adjusted
https://drive.google.com/open?id=1KIrXvQDkLMhOtZsShzoAzRlx8L89DPys

**Comment by Unre-Alex** [comment]
That's more like it, but now the air control needs to be adjusted for better comparison (it should move faster from side to side). Did you change something with the projectiles' speed? In UT3 they are faster.

**Comment by HellDragon-HK** [comment]
I changed the projectile to the same speed value UT3 has and yes I agree UT3 still seems faster, how much faster do you think, I don't think it's too far off now and I'll try messing with air control now but remember UT3 Manta has an air speed setting UT2004 doesn't and I don't think it can

**Comment by Unre-Alex** [comment]
The driver position is fine.  The projectiles need to be slightly faster and the damage reduced to 25-6-7?

**Comment by HellDragon-HK** [comment]
Ok so we at least can say we improved the driver pos, did you try that damage value yet yourself?
Still playing around with air control and I think it's the fact that it takes the UT2004 Manta so long to stop moving and change direction that is the issue because UT3 Manta from my testing stops on a dime basically

**Comment by Unre-Alex** [comment]
Yeap, in UT3 it's 25.

**Comment by HellDragon-HK** [comment]
? Then what is this, is the code GE has outdated or something ?
http://greatemerald.xmpcommunity.com/uncodex/UT3-15/utgamecontent/utproj_mantabolt.html

**Comment by Unre-Alex** [comment]
I changed it to 25 and in UT2k4 it takes like a second longer to destroy another Manta.

**Comment by HellDragon-HK** [comment]
Not doubting you just dumbfounded it's so different of a value although speed is proving that way as well as I'm at 2200 at it still feels slow, I'm going to try 3000 next

**Comment by HellDragon-HK** [comment]
@Unre-Alex 3000 feels about right, acceleration may need adjusting still I also think the shots in UT3 have a longer distance because I can see the shots hitting way up on the mountains in Suspense but on DryIce I have to aim lower on the mountains which look a lot closer....what do you think?

**Comment by Unre-Alex** [comment]
3000 feels right but the sound is slower?

**Comment by HellDragon-HK** [comment]
I know but what can be done about it, speed up the sound file?

**Comment by HellDragon-HK** [comment]
@Unre-Alex What's your take on max range (LifeSpan), is 1.6 good or does it feel like it has more distance in UT3 and needs to be around 1.8 or 1.7 maybe?

**Comment by Unre-Alex** [comment]
It's more for sure, but how much you tell me and I'll test it.

**Comment by HellDragon-HK** [comment]
1.8 is my guess but that might be too much, I looked closer and in UT3 the shots are exploding mid air so they are not actually making it to their target

**Comment by HellDragon-HK** [comment]
@Unre-Alex Try this for turning and see what you think, I think it's almost there
    TurnTorqueMax=145.0

**Comment by Unre-Alex** [comment]
1.8 (1.75 seems fine too) and the turning in UT3 feels faster, but probably because of the air control.

**Comment by HellDragon-HK** [comment]
I have the air control at 18.0 on mine and it doesn't really feel like it does much but give it a try, and we can keep tweaking the turn speed I think, let's try 160 maybe?

**Comment by HellDragon-HK** [comment]
@Unre-Alex 
What about Ground Speed change from 2000 to 1500, it seems right to me but what do you think?

**Comment by Unre-Alex** [comment]
Changing the air control value doesn't do a thing, without it it's hard to make a judgment on the other stuff. I'm done for today.

**Comment by HellDragon-HK** [comment]
@Unre-Alex Alright, thanks for the help in figuring things out, a little closer to a real UT3 Manta not just in look but the way it plays :D

**Comment by HellDragon-HK** [comment]
Well I got a headlight projector to work on the Manta but we don't have a texture for it's light type

**Comment by HellDragon-HK** [comment]
What? What kind of errors, like crash/CTD type errors?  I definitely have not been experiencing that, as it's just us doing the testing it seems can you tell me what you've tried and where so I can try to replicate it?

**Comment by Unre-Alex** [comment]
When I add MaxGroundSpeed for the Manta it doesn't compile.

**Comment by HellDragon-HK** [comment]
I don't think MaxGroundSpeed works on many vehicles but I think all use GroundSpeed

**Comment by GreatEmerald** [comment]
Looking at the code, `MaxGroundSpeed` is implemented manually for the Goliath for instance. But it merely changes `KarmaParams`, `kMaxSpeed`, so you can just use that directly: http://greatemerald.xmpcommunity.com/uncodex/UT2004/Source_onslaught/onshovertank.html#385

**Comment by HellDragon-HK** [comment]
You mean use kMaxSpeed to set max speed? If yes then that should help solve some issues...assuming it will accept it on compile which is quite touchy

**Comment by HellDragon-HK** [comment]
Tried to do team coronas and had slight success....namely only making them appear...all the time and never disappear, any attempt to add Destroyed and the Manta lost all of it's movement properties in-game, either someone else does it or tell me exactly what I need to do because I don't get it no matter what other codes I look at

/*simulated function PostNetBeginPlay() //Someone else is going to have to make this work right, it shows but is always showing no matter what even if dead
*{
*    local int i;
*
*    Super.PostNetBeginPlay();
*
*	if(Level.NetMode != NM_DedicatedServer && Level.bUseHeadlights && !(Level.bDropDetail || (Level.DetailMode == DM_Low)))
*	{
*		HeadlightCorona.Length = HeadlightCoronaOffset.Length;
*
*		for(i=0; i<HeadlightCoronaOffset.Length; i++)
*		{
*			HeadlightCorona[i] = spawn( class'UT3HeadlightCorona', self,, Location + (HeadlightCoronaOffset[i] >> Rotation) );
*           HeadlightCorona[i].SetBase(self);
*			HeadlightCorona[i].SetRelativeRotation(rot(0,0,0));
*			HeadlightCorona[i].Skins[0] = HeadlightCoronaMaterial;
*			HeadlightCorona[i].ChangeTeamTint(Team);
*			HeadlightCorona[i].MaxCoronaSize = HeadlightCoronaMaxSize * Level.HeadlightScaling;
*		}
*    }
*    SetTeamNum(Team);
*	TeamChanged();
}*/

**Comment by 100GPing100** [comment]
@HellDragon-HK You have to destroy them when the vehicle is destroyed, just like the smoke emitters.

**Comment by HellDragon-HK** [comment]
I added it to the Function Destroy area as well as added in the Destroy Appearance, but I don't know enough to understand it and strangely it removed the Manta's properties as I kept trying even though the defaultproperties were not touched

**Comment by GreatEmerald** [comment]
Note: to format code blocks, place "```UnrealScript" at the beginning of the code block,

and place "```" at the end of it.

**Comment by HellDragon-HK** [comment]
I don't see that anywhere in the codes I looked at GE, and I copied it straight from ONSVehicle.uc minus the unneeded bits about the weapons & sparks.

**Comment by HellDragon-HK** [comment]
I think I'm ready to upload the Manta animation binary, it's been doing well with the only oddity being the death mesh, which I'm not sure where that's changed but I believe it's in both code and animation isn't it?

Anyway @Unre-Alex @100GPing100 what do you guys think, good enough yet?

**Comment by 100GPing100** [comment]
@HellDragon-HK GreatEmerald meant this [here](https://help.github.com/articles/creating-and-highlighting-code-blocks/).

As for testing, I still haven't looked into manta stuff.

**Comment by HellDragon-HK** [comment]
Ah ok so it's for this site to make it easier to read then, I misunderstood

**Comment by GreatEmerald** [comment]
Yes. Manta does not have the DrawScale set, which means it inherits it from the closest parent that does, which happens to be `Actor` (1.0).

**Comment by HellDragon-HK** [comment]
On network game blades are always spinning even on initial spawn before game starts

**Comment by HellDragon-HK** [comment]
@100GPing100 Well it definitely stops on a dime now, interestingly though it takes slightly longer to stop by holding back now which is weird

Engine sound is still too high pitched, it's not just the movement, in UT3 just sitting still is even different sounding

Also @GreatEmerald since ````MaxPitchSpeed =```` is sound related apparently shouldn't it be moved to out of 'Movement' to where the sound data is to avoid confusion?

**Comment by GreatEmerald** [comment]
Hm, looking at UT3 source, it also has engine sounds changing pitch when moving. I'm not sure why there would be a difference between the games here.

Sure, if it's sound related it should be next to sound options.

**Comment by HellDragon-HK** [comment]
It does change pitch 'while moving' in UT3 but the engine pitch itself is much lower in UT3 right from the start, it sounds very different and it's really quiet when just sitting still

**Comment by GreatEmerald** [comment]
Hm, that might be possible to change by changing the actual pitch of the sound and the volume. In UT3 this is probably handled in native code, and I guess in UT2004 as well, so we can't do it in code.

**Comment by HellDragon-HK** [comment]
So you mean tamper with the actual file until it sounds right?  That might be better left to someone else, I don't know much about that field.

**Comment by HellDragon-HK** [comment]
Headlight placement should be fixed now, and the engine flames are done but an unsolved issue has cropped up I need help solving

#188 
#187

**Comment by HellDragon-HK** [comment]
Just uploaded the Manta movement vid, all of them have been done since I mentioned even doing them but I just wasn't sure if they would be very useful but since @100GPing100 doesn't have access to UT3 right now it probably wouldn't hurt to have any kind of reference possible....that said all vehicles including those not yet in the mod have a vid (except turrets) and I'll see about getting those uploaded, if needed I can upload them to YT instead probably.
